src/track.js

```diff
@@ -144,6 +145,23 @@ function extractIGCTracks(igc) {
   return points.length > 0 ? [{timestamp, points, name}] : [];
 }
 
+function extractSKIZTracks(skiz) {
+    const points = [];
+    let timestamp = null;
+
+    for (const node of skiz.trackNodes) {
+        points.push({
+            lat: node.latitude,
+            lng: node.longitude,
+        });
+
+        node.timestamp && (timestamp = node.timestamp);
+    }
+
+    const name = 'skiz';
+    return points.length > 0 ? [{timestamp, points, name}] : []
```

Comment:
tiny nit, could you add a semicolon here? below on 246 as well

Reply:
done!